lambda/src/main/scala/pricemigrationengine/services/Zuora.scala

```diff
@@ -45,4 +49,13 @@ object Zuora {
 
   def renewSubscription(subscriptionNumber: String): ZIO[Zuora, ZuoraRenewalFailure, Unit] =
     ZIO.environmentWithZIO(_.get.renewSubscription(subscriptionNumber))
+
+  // Note: the Zuora documentation
+  // https://www.zuora.com/developer/api-references/older-api/operation/GET_AmendmentsBySubscriptionID/
+  // specifies that a subscriptionId is to be provided, but it also works with a subscription number
+  // (aka subscription name for a cohort item).
```

Comment:
I think Zuora also refers to subscription numbers as subscription names in some of their docs as well!  (And also possibly the UI as well somewhere)